### DISCUSSION TITLE
希望增加我的页面中的服务自定义

### DISCUSSION THREAD

**Discussion by nathalie-annis** [BiliUniverse/.github#4] [Q&A 问答] [answered]
希望能够添加服务功能的自定义，更有利于个性化，比如偶尔用的到会员购，又不想底栏一直显示会员购，就可以在服务功能中开启显示。

**Comment by VirgilClyne** [comment, marked as answer]
可以使用 [BoxJs](https://biliuniverse.io/guide/box-js) 进行详细的设置
